service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -160,8 +167,10 @@ public void addDirectionsToCaseDetails(CaseDetails caseDetails,
                 case PARENTS_AND_RESPONDENTS:
                     directions.addAll(clone);
 
-                    if (eventId == COMPLY_ON_BEHALF_SDO) {
+                    if (eventId == COMPLY_ON_BEHALF_COURT) {
                         filterResponsesNotCompliedOnBehalfOfByTheCourt("RESPONDENT", directions);
+
+                        directions = getDirectionsForUnrepresentedParties(caseDetails, directions);
```

Comment:
We need to run similar logic for others as well, don't we?

Reply:
yep! Just not implemented yet